experimental/oak_baremetal_channel/schema.fbs

```diff
@@ -22,4 +23,13 @@ table Initialization {
     wasm_module: [ubyte];
 }
 
+table LookupDataEntry {
+    key: [ubyte];
+    value: [ubyte];
+}
+
+table LookupData {
+    items: [LookupDataEntry];
+}
```

Comment:
Later on we may want to add methods to build up a LookupData file incrementally, so parsing doesn't happen all at once

Reply:
Yes, I agree.